### ISSUE TITLE
GPS coordinates not consistently validated in bulk import

### ISSUE THREAD

**Issue by goddesswarship** [WildMeOrg/Wildbook#751]
**Expected behavior**
If encounters are missing data under a field, the bulk import review page (`/import/standard-upload`) should flag those cells in blue.

**Current behavior**
If encounters are missing data under `Encounter.decimalLatitude` and `Encounter.decimalLongitude` fields AND 
`Encounter.genus`  and `Encounter.specificEpithet`, the error message displayed is:
`Errors are preventing the submission of this bulk import`

If only some encounters under the lat/long fields are missing data, they are converted from blank cells to `0.0`

**Testing notes**
The result is the same if the lat/long field cells are changed from Numbers to General or Text. Spreadsheets for testing are in the services inbox with the same subject as the Community post.

**Community link**
https://community.wildme.org/t/strange-bulk-import-error-messaging/3220

**Comment by TanyaStere42** [comment]
If this only happens when species info is missing, then the error is correct, but the highlighting isn't. You can't bulk submit without species on Wildbook

**Comment by goddesswarship** [comment]
Right. It's the inconsistent flagging the lat/long fields between the two use cases that was initially raised. When species is missing it also throws in a "by the way, double-check these other fields, too." But if it's just some lat/long entries missing, it's not flagged at all. This could potentially be rolled into this other issue on bulk import not validating all fields https://github.com/WildMeOrg/Wildbook/issues/317. Let me know what you think.

**Comment by TanyaStere42** [comment]
Gotcha. I think it makes sense as a separate ticket, but we should get rid of the info about the error. That behavior is as expected, and it seems like that's the problem with it in the unexpected behavior section

**Comment by naknomum** [comment]
these are now being validated in bulk import (both FE and BE)
